compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -1799,80 +1805,66 @@ OMR::ResolvedMethodSymbol::getOSRTransitionTreeTop(TR::TreeTop *tt)
    }
 
 void
-OMR::ResolvedMethodSymbol::insertStoresForDeadStackSlotsBeforeInducingOSR(TR::Compilation *comp, int32_t inlinedSiteIndex, TR_ByteCodeInfo &byteCodeInfo, TR::TreeTop *induceOSRTree, TR::ResolvedMethodSymbol *callSymbolForDeadSlots)
+OMR::ResolvedMethodSymbol::insertStoresForDeadStackSlots(TR::Compilation *comp, TR_ByteCodeInfo &byteCodeInfo, TR::TreeTop *insertTree, bool ignoreStashedArgs)
```

Comment:
Can you discuss the need for this new bool parameters and what is signifies? A the very least this needs some doxygen to describe how it is used.